Documentation/learning/glossary.md

```diff
@@ -26,10 +30,93 @@ Peer is another member of the same cluster.
 
 A proposal is a request (for example a write request, a configuration change request) that needs to go through raft protocol.
 
+## Quorum
+
+The number of members of a group/cluster required to be present to perform a set of instructions/operations, usually a majority.
+
 ## Client
 
 Client is a caller of the cluster's HTTP API.
 
-## Machine (deprecated)
+## Command
 
-The alternative of Member in etcd before 2.0
+Command is an identifiers which are used to perform a specific operation in etcd server using etcdctl.
+Example: put, get, del, lease, etc.
+
+## Transaction
+
+Transaction is a set of instructions which are to be performed as an atomic operation based on a certain set of conditions.
```

Comment:
Transaction is a set of operations that are to be executed atomically based on a certain set of conditions.